tool/lib/commands/generate_changelog.dart

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert' show jsonDecode;
+import 'dart:io';
+
+import 'package:http/http.dart' as http;
+
+import 'package:args/command_runner.dart';
+
+import '../model.dart';
+
+class GenerateChangelogCommand extends Command {
```

Comment:
please add an example of calling this command above the class

Reply:
Done, good idea!